app/containers/HeaderGroupChatContainer/HeaderGroupChatContainer.tsx

```diff
@@ -0,0 +1,44 @@
+import { Text, Image, TouchableOpacity, View } from 'react-native';
+import { useSelector } from 'react-redux';
+
+import { UserStatus } from '@Models/index';
+import { currentGroupSelector } from '@Stores/groups';
+import { getUserStatusByIdSelector, userIdSelector } from '@Stores/user';
+import { getImageSource } from '@Utils/index';
+
+import { styles } from './HeaderGroupChatContainerStyles';
+
+interface HeaderGroupChatContainerProps {
+  onClickHeader?: () => void;
+}
+
+export const HeaderGroupChatContainer = (props: HeaderGroupChatContainerProps) => {
+  const { onClickHeader } = props;
+
+  const currentGroup = useSelector(currentGroupSelector);
+  const userId = useSelector(userIdSelector);
+  const isMoreThan2Member = !!currentGroup && currentGroup.members.length > 2;
+
+  const otherMember = currentGroup?.members.filter((member) => member._id !== userId);
+
+  const userStatusSelector = useSelector(getUserStatusByIdSelector);
+  const userStatus = userStatusSelector({ userId: otherMember ? otherMember[0]._id : '' });
+
+  if (!currentGroup) {
+    return <></>;
+  }
+
+  return (
+    <TouchableOpacity onPress={onClickHeader} style={styles.container}>
```

Comment:
Hung, what if we don't pass onClickHeader, the button should be disabled, am I right?
You can use `disabled = { !onClickHeader } `